helpers/notification_helper.py

```diff
@@ -155,17 +155,9 @@ def send_broadcast(cls, client_types, title, message, url, app_version=''):
 
     @classmethod
     def send_ping(cls, client):
-        if client.client_type == ClientType.WEBHOOK:
-            keys = {client.client_type: [(client.messaging_id, client.secret)]}
+        if client.client_type == ClientType.OS_ANDROID:
```

Comment:
Moving forward - this `NotificationHelper` will exist and act as a router to figure out if we should dispatch the old notifications for Android, or the new notifications for other platforms. Once we can get Android moved over to the new notification system, we should be able to remove this code. But for now, not breaking Android is a priority.